gradle-jdks-setup/src/main/resources/gradle-jdks-setup.sh

```diff
@@ -158,6 +176,7 @@ done
 
 rm -rf "$tmp_work_dir"
 
+# TODO(crogoz): what happens if gradle-daemon-jdk-version doesn't exist or the jdk for that version doesn't exist ?
 gradle_daemon_jdk_version=$(read_value "$APP_GRADLE_DIR"/gradle-daemon-jdk-version)
```

Comment:
if `gradle-daemon-jdk-version` is not present or the JDK version specified does not exist in gradle/jdks (https://github.com/palantir/gradle-jdks/pull/360/files#diff-b43a6a48890fa88a6f76f47d3b3ac00c70432c5451f9551ee0d81e1f5db6ec7eR181), then `./gradlew task` will fail. I'll add a FLUP PR for these cases.